commons/all/src/main/java/org/infinispan/commons/configuration/io/yaml/YamlConfigurationWriter.java

```diff
@@ -199,6 +199,20 @@ public void writeAttribute(String name, Iterable<String> values) {
       }
    }
 
+   private static String escapeYAML(String s) {
```

Comment:
Seems to be missing a call in `#writeCharacters(String)` at line 245 with the diff in.

Reply:
Done